Makefile

```diff
@@ -139,13 +142,61 @@ help-install:
 #
 ###############################################################
 
+### Deployment ###############################################
+#
+release-check:
+	@VERSION=$(shell sh -c "git describe --exact") && \
+	 if test -z "$$VERSION"; then                     \
+		echo VERSION is not set.;                 \
+		echo Please create a git tag;             \
+		exit 100;                                 \
+	else                                              \
+		echo "Releasing version: $$VERSION";      \
+	fi;
+
+release-prep: release-check
+	@VERSION=$(shell sh -c "git describe --exact") && \
+	python setup.py clean;                            \
+	python setup.py sdist;                            \
+	pip install dist/starfish-$$VERSION.tar.gz
+
+release-verify: release-check slow
```

Comment:
This doesn't install your to-be-released version, right?  So this is checking source against the notebooks?  Would it be preferable to use the environment prepped in `release-prep` and run it there?

Reply:
Prep does. At the moment, it's very much setup to use the three in order. `prep` and `verify` can be combined if you'd prefer.

Reply:
Would it make sense to make `release-verify` explicitly depend on `release-prep` then?

Reply:
Happy either way. If everyone prefers, maybe it's just `release-build` and `release-publish`.